src/web/backend/nextcloud/utils.py

```diff
@@ -0,0 +1,8 @@
+
+
+def edap_to_dict(obj):
```

Comment:
As I have said earlier, this is a great concept, and I think that you can go further - introduce `edap_users_to_list` or something like that.

Reply:
+ will add in separate PR